lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -89,7 +89,7 @@
           percentage="main.state.getFlashState().percentage"
           striped="{{ main.state.getFlashState().type == 'check' }}"
           ng-attr-active="{{ main.state.isFlashing() }}"
-          ng-click="flash.flashImageToDrive(main.selection.getImagePath(), main.selection.getDrive())"
+          ng-click="flash.flashImageToDrive(main.selection.getImage(), main.selection.getDrive())"
```

Comment:
I don't really have something against it. Or I can put them in a different commit. But I want to see if @jviotti agrees with the approach of giving the whole `image` to the `flashImageToDrive`.